### PR TITLE
#167980341 User can vote for or against a petition

### PR DESCRIPTION
**Pull request by Jordybastien**
#### What does this PR do?
This PR adds functionality to ensure that a user can vote for or against a petition
#### Description of Task to be completed?
Add Vote service to link to the database, a vote controller and routes, token authentication is pending
#### How should this be manually tested?
Review the source code
#### Any background context you want to provide?
This work is still in progress and not has been made for other members to pick up the project flow
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167980341
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by mucyomiller — APPROVED** [review]
LGTM

File: src/controllers/voteController.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        if (!req.body.user_id || !req.body.vote || !req.params) {
+          util.setError(400, 'Please provide complete details');
+          return util.send(res);
+        }
+        // eslint-disable-next-line camelcase
+        const { vote, user_id } = req.body;
+        // userId get from token
+        const newId = req.params;
+        // eslint-disable-next-line camelcase
+        const petition_id = newId.id;
```
Comment (amilykassim) [inline src/controllers/voteController.js:21]: Everything is good, but it seems you didn't destructure the id from the req.params and it caused you to end up writing redundant codes.

**Review by amilykassim — APPROVED** [review]

File: src/controllers/voteController.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable indent */
+import voteService from '../services/VoteService';
+import PetitionService from '../services/PetitionServices';
+import Util from '../utils/Utils';
+
+const util = new Util();
+
+class voteController {
```
Comment (gik-hub) [inline src/controllers/voteController.js:9]: Es6 classes should start with a capital letter

File: src/controllers/voteController.js (hunk 1 above)
Comment (Jordybastien) [reply src/controllers/voteController.js:21]: let me take a look at it

**Review by gik-hub — COMMENTED** [review]
Please consider following the standard naming conventions in the final PR you will raise

**Pull request merged**